crates/bevy_render/src/renderer/mod.rs

```diff
@@ -141,13 +141,21 @@ impl<T> WgpuWrapper<T> {
     pub fn new(t: T) -> Self {
         Self(t)
     }
+
+    pub fn into_inner(self) -> T {
+        self.0
+    }
 }
 
 #[cfg(all(target_arch = "wasm32", target_feature = "atomics"))]
 impl<T> WgpuWrapper<T> {
     pub fn new(t: T) -> Self {
         Self(send_wrapper::SendWrapper::new(t))
     }
+
+    pub fn into_inner(self) -> T {
+        self.0.take()
```

Comment:
This is only safe from the thread that created the WgpuWrapper, and will crash otherwise, according to the documentation.